albumentations/augmentations/transforms.py

```diff
@@ -1064,12 +1065,14 @@ def get_transform_init_args_names(self):
                 'min_height', 'min_width')
 
 
-class JpegCompression(ImageOnlyTransform):
```

Comment:
we cannot change api, see how we deprecate things in `RandomBrightness` and `RandomContrast`

Reply:
I'm so sorry. It's my mistake. I fixed that.